### PR TITLE
ML image analysis project

### PR DESCRIPTION
**Pull request by gcheng21**
This project uses the Cloud Vision API to use machine learning to extract labels from images uploaded by users in a message

### CODE REVIEW

File: bin/.gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+target
```
Comment (elithrar) [inline bin/.gitignore:1]: Remove this file - I assume you added by accident.

File: bin/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# How to Contribute
```
Comment (elithrar) [inline bin/CONTRIBUTING.md:1]: Remove this file - I assume you added by accident.

File: bin/LICENSE [hunk 3]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (elithrar) [inline bin/LICENSE:1]: Remove this file - I assume you added by accident.

**Review by elithrar — CHANGES_REQUESTED** [review]
Hey @gcheng21 - looks like you may have accidentally committed a copy of the repo into the `/bin` dir - can you reset the commit and fix?

**Comment by gcheng21** [comment]
> Hey @gcheng21 - looks like you may have accidentally committed a copy of the repo into the `/bin` dir - can you reset the commit and fix?

@elithrar  how do i reset a commit

**Comment by elithrar** [comment]
@gcheng21 

In the branch you're working on:

```
git reset HEAD
```

... will "unstage" the last commit.

```
git reset HEAD~N
```

... will unstage the last N commits. Also see https://stackoverflow.com/questions/927358/how-do-i-undo-the-most-recent-commits-in-git - let me know it's still being a pain!

**Pull request closed without merging**